Docs/source/usage/parameters.rst

```diff
@@ -81,6 +81,38 @@ Overall simulation parameters
     one should not expect to obtain the same random numbers,
     even if a fixed ``warpx.random_seed`` is provided.
 
+* ``algo.evolve_scheme`` (`string`, default: `explicit`)
+    Specifies the evolve scheme used by WarpX.
+
+    * ``explicit``: Use an explicit solver, such as the standard FDTD or PSATD
+
+    * ``implicit_picard``: Use an energy conserving implicit solver that uses a Picard iteration to solve the system.
+      Note that this method is for demonstration only. It is inefficient and does not work well when
+      :math:`\omega_{pe} \delta t` is close to or greater than one.
```

Comment:
```suggestion
      :math:`\omega_{pe} \Delta t` is close to or greater than one.
```
I think that we use the notation `\Delta t` rather than `\delta t` in the rest of the documentation.